lerna.json

```diff
@@ -1,5 +1,4 @@
 {
   "version": "0.6.0",
-  "npmClient": "yarn",
-  "useWorkspaces": true
```

Comment:
Don't we use lerna workspaces?

Reply:
https://github.com/lerna/lerna/pull/3695
lerna removed the useWorkspaces, and makes it the default.